content/strategy-goals/strategy/code-insights/index.md

```diff
@@ -24,14 +24,15 @@ We bring **the best practices of big data to challenges of big code**, turning a
 
 **We help engineering leaders** set and measure quantitative engineering goals, whether they be related to a migration, deprecation, vulnerability, code smell, or high level understanding. We also help engineering leaders understand the shape of their code and how it changes over time. Then we help them tie trends and metrics to _owners_ on their engineering teams.
 
-**We help individual engineers** measure the day-to-day goals of their teams – how they are progressing on an individual initiative, how quickly other teams are adopting their code – as well as quantify tech debt problems to justify to leaders why certain work is important. We also help them get a historical usage view of functions, changes, or other patterns. Happily, we've heard that Code Insights has even helped some engineers get promotions or bonuses, because they use code insights to quantify the impact and spread of their work.
+**We help individual engineers** measure the day-to-day goals of their teams – how they are progressing on an individual initiative, how quickly other teams are adopting their code – as well as quantify tech debt problems to justify to leaders why certain work is important. We also help give them a high level or historical usage view of functions, changes, or other patterns that improve their core Sourcegraph use of searching and navigating code. Happily, we've heard that Code Insights has even helped some engineers get promotions or bonuses, because they use code insights to quantify the impact and spread of their work.
 
 ### Strategy
 
 To deliver on our mission, we will:
 
-- First, build a robust Code Insights product offering that is configurable yet simple enough to answer many different engineering leaders' specific questions
-- Continually track usage and feedback to grow the product in its most valuable directions
+- ✅ First, build a robust Code Insights product offering that is configurable yet simple enough to answer many different engineering leaders' specific questions
+- 🔄 Continually track usage and feedback to grow the product in its most valuable directions
+- Now, build Code Insights-like features that empower the core workflow of individual developers who use Sourcegraph 10+ times a day to search and navigate code
```

Comment:
@Joelkw maybe code insights-powered features? Code-insights-like is a little confusing (why do we need a 'code-insights-like' if we have code insights. Small change thought

Reply:
Thanks, will revert